### ISSUE TITLE
Add Readme and Installation Steps

### ISSUE THREAD

**Issue by bepitulaz** [froyostory/story-cms#11]
Please add README.md and the installation steps. I want to know how to install this CMS. Thanks.

**Comment by purwandi** [comment]
Good point, we will add instruction how to install in when the code ready to public used.
